### PR TITLE
Updated tutorial-install-ubuntu-desktop for 18.04 (DON'T PUBLISH YET?)

### LINKED ISSUE #713: Update installing ubuntu desktop for 18.04

### PR DESCRIPTION
**Pull request by degville**
fixes #713 

Also moves the old tutorial to tutorial-install-ubuntu-desktop-1604 so that it's still available.

### CODE REVIEW

File: tutorials/desktop/install-ubuntu-desktop/install-ubuntu-desktop-1804.md [hunk 1]
```diff
@@ -0,0 +1,154 @@
...
+You can also choose to enable automatic login and home folder encryption. If your machine is portable, we recommend keeping automatic login disabled and enabling encryption. This should stop people accessing your personal files if the machine is lost or stolen.
+
+![Username and password entry](https://assets.ubuntu.com/v1/abd7511b-bionic-install-password.png)
+
+: If you enable home folder encryption and you forget your password, you won't be able to retrieve any personal data stored in your home folder.
+
+## Background installation
+Duration: 0:20
+
+The installer will now complete in the background while the installation window teaches you a little about how awesome Ubuntu is. Depending on the speed of your machine and network connection, this should only take a few minutes.
```
Comment (evilnick) [inline tutorials/desktop/install-ubuntu-desktop/install-ubuntu-desktop-1804.md]: surely it takes ages to explain how awesome Ubuntu is :)

**Comment by evilnick** [comment]
Oh, and change the dates!

**Comment by anthonydillon** [comment]
Does this block updates to tutorials until release? If so, I would recommend sending the team an email so noone does release without knowing.

**Comment by degville** [comment]
@anthonydillon After discussion, we decided it would be acceptable if this was published early; the old version is still accessible, there are already update guides for Bionic and we want ours to be canonical, and this lets us test and share the tutorial.

**Comment by anthonydillon** [comment]
Thanks for the update. Make's sense.

**Pull request merged**